### DISCUSSION TITLE
Expo - disable analytics

### DISCUSSION THREAD

**Discussion by CallbackDan** [invertase/react-native-firebase#8072] [Q&A]
Hi- is there a way to disable analytics when using expo and the analytics plugin? 

Prior to using Expo this was the one:

{
  "react-native": {
    "analytics_auto_collection_enabled": false
  }
}

**Comment by mikehardy** [comment]
You can still have a firebase.json even while using expo, correct? I assume so, which means you would look at the schema for firebase.json and use the keys that match your use case https://rnfirebase.io/app/json-config#json-schema ?
